src/chromo/models/qgsjet.py

```diff
@@ -134,11 +136,18 @@ class QGSJet2Run(QGSJetRun):
 
     def _cross_section(self, kin=None):
         kin = self.kinematics if kin is None else kin
-
+        _projectile_id = {
```

Comment:
It is not pythonic to start local variables with an underscore. These variables are not visible outside of the scope of the function anyway. We only use the underscore for attributes and methods, where it gets special treatment.

Reply:
Cheers. I had removed the underscores in some other places following this idiom but forgot it here. Is now fixed.